main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -1931,4 +1936,17 @@ public static int getListInitialLoadLimit() {
         return sensitiveKeys;
     }
 
+    public static void putLogTemplate(final String preferenceBaseName, final String title, final String text) {
+        putStringDirect(preferenceBaseName + ".Title", title);
+        putStringDirect(preferenceBaseName + ".Text", text);
+        putStringDirect(preferenceBaseName, title + ": " + text);
```

Comment:
Why is this third property needed?

Reply:
.Title and .Text are what's used for editing and inserting. "preferenceBaseName" is what's shown in the preferences dialog - that one can display only 1 value and that one must be a preference. So I can either display "just title", "just text" or this combined preference there.

![image](https://user-images.githubusercontent.com/1258173/141003041-ab5f032e-bf2f-43b9-85a4-84be52bc88e3.png)